docs/community/release_guide.md

```diff
@@ -175,6 +175,45 @@ After cutting release branch, don't forget bump version in `main` branch.
 export RELEASE_RC_NO=<RC number, e.g. 0>
 ```
 
+- Check License and Notice
+
+Build a release package and check license and notice with the binary package's jars directory instead of the pom.xml.
+```shell
+./buile/make-distribution.sh --release --sbt-enabled
+```
+According to the Apache License section 4, the principal of License and Notice is as following:
```

Comment:
can we add the hyperlink for "Apache License section 4", and use quote syntax for the reference sections

Reply:
Added the hyperlink.
The following text is not a reference, it's a conclusion.
Ping @pan3793